folding/src/lib.rs

```diff
@@ -237,6 +237,10 @@ impl<'a, CF: FoldingConfig> FoldingScheme<'a, CF> {
         fq_sponge.absorb_g(&error_commitments[0].elems);
         fq_sponge.absorb_g(&error_commitments[1].elems);
 
+        let to_absorb = env.to_absorb();
```

Comment:
Can you indicate the order of the absorbtion in the documentation, please? We need to be consistent for the IVC circuit, and it is good to document it.
We should have a test checking the order is in line with the documentation.

Reply:
Note that the typed transcript could be used, see https://github.com/o1-labs/proof-systems/pull/2094.
The idea is that every "round" (compute_error, compute_extended, etc) sends a message type, that can be absorb.
Each function returns a type "Message", and the sponge gets to absorb a heterogeneous list of "absorbable messages".
The heterogeneous list can be implemented as `Nil, Cons` constructors.

But I suppose it will be for the future. For now, we can keep it simple, and write in comments the order.